test/e2e/network/networking.go

```diff
@@ -48,13 +48,13 @@ var _ = SIGDescribe("Networking", func() {
 	ginkgo.It("should provide Internet connection for containers [Feature:Networking-IPv4]", func() {
 		ginkgo.By("Running container which tries to ping 8.8.8.8")
 		framework.ExpectNoError(
-			framework.CheckConnectivityToHost(f, "", "ping-test", "8.8.8.8", framework.IPv4PingCommand, 30))
+			framework.CheckConnectivityToHost(f, "", "ping-test", "8.8.8.8", framework.IPv4, 30))
```

Comment:
In the description you mentioned:

>This PR updates the networking e2e test on Azure to use nc instead of ping for IPv4 tests. This is because Azure's LB drops ICMP packets for security reasons.

But this test is only validating that pod inside the cluster can ping an external IP, nothing goes through LB unless I'm missing something

Reply:
Oh, is it because you want to use `CheckConnectivityToHost` in your custom Azure tests? Can you just create a new method local to your tests in that case?